### ISSUE TITLE
Artifact storage raising exceptions

### ISSUE THREAD

**Issue by taquitos** [fastlane/ci#380]
form pry session:
```
[5] pry(#<FastlaneCI::LocalArtifactProvider>)> File.exist?(original_artifact_reference)
false
[6] pry(#<FastlaneCI::LocalArtifactProvider>)> original_artifact_reference
=> #<Pathname:fastlane.log>ArtifactProvider>)> 
```

**Comment by taquitos** [comment]
Seems that it doesn't like the fastlane.log 🤷‍♂️

**Comment by minuscorp** [comment]
Fixed by #383

**Comment by KrauseFx** [comment]
Nice 👍
